stories/features/rewards/settingsMobile/style.ts

```diff
@@ -0,0 +1,68 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License. v. 2.0. If a copy of the MPL was not distributed with this file.
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import styled from 'styled-components'
+
+export const StyledListContent = styled<{}, 'div'>('div')`
+  padding: 0 25px;
+`
+
+export const StyledSupport = styled<{}, 'div'>('div')`
+  background: #F2F4F7;
```

Comment:
Colors should at least use palette variables, if we can't come up with theme variables. Can you use them where you can, or confirm with design if they are adding colors to their palette?